src/test/java/online/partyrun/partyrunauthenticationservice/domain/auth/repository/RefreshTokenRepositoryTest.java

```diff
@@ -0,0 +1,70 @@
+package online.partyrun.partyrunauthenticationservice.domain.auth.repository;
+
+import online.partyrun.partyrunauthenticationservice.TestConfig;
+import online.partyrun.testmanager.redis.EnableRedisTest;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Import;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+@EnableRedisTest
+@Import(TestConfig.class)
+@DisplayName("RefreshTokenRepository")
+class RefreshTokenRepositoryTest {
+
+    @Autowired
+    RefreshTokenRepository refreshTokenRepository;
+
+    @Value("${jwt.refresh-expire-second}")
+    private long refreshExpireSecond;
+    String name = "박성우";
+    String refreshToken = "refreshToken";
+
+    @Nested
+    @DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+    class set은 {
+
+        @Test
+        @DisplayName("RefreshToken이 이미 저장되어있다면, .")
+        void setRefreshToken() {
+
+        }
+    }
+
+    @Nested
+    @DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+    class existsBy는 {
+
+        @Test
+        @DisplayName("RefreshToken이 저장되었는지 확인한다.")
+        void exists() {
+            refreshTokenRepository.set(name, refreshToken);
+
+            assertThat(refreshTokenRepository.existsBy(name, refreshToken)).isTrue();
+        }
+
+        @Test
+        @DisplayName("시간 초과 이후 조회하면 false를 반환한다.")
+        void notExistsByExpireTime() throws InterruptedException {
+            refreshTokenRepository.set(name, refreshToken);
+            Thread.sleep(refreshExpireSecond * 1000);
```

Comment:
```suggestion
@Test
        @DisplayName("시간 초과 이후 조회하면 false를 반환한다.")
        void notExistsByExpireTime() {
            ScheduledExecutorService executorService = Executors.newSingleThreadScheduledExecutor();
            refreshTokenRepository.set(name, refreshToken);
            executorService.schedule(() -> {
                assertThat(refreshTokenRepository.existsBy(name, refreshToken)).isFalse();
            }, refreshExpireSecond, TimeUnit.SECONDS);
        }
```

이런 식으로 작성하면 Thread.sleep 을 사용하지 않고 테스트 가능한데 어떠신가요?? 아래 실행 시간 이미지 첨부할게요!

![image](https://github.com/SWM-KAWAI-MANS/party-run-authentication-service/assets/55674648/dd270691-88e4-4e39-a506-7cfc0318f989)